crates/host/src/wasmbus/mod.rs

```diff
@@ -70,6 +58,17 @@ use wasmcloud_runtime::capability::{
 use wasmcloud_runtime::Runtime;
 use wasmcloud_tracing::context::TraceContextInjector;
 
+use crate::{
```

Comment:
Why the reorder here?

Reply:
It was automatic, maybe new rust-analyzer? I can move back but this seems closer in line with the convention we use, being
1. core
2. std
3. external
4. internal

Reply:
The order I saw elsewhere was
1. internal (`crate`)
2. core
3. std
4. external

FWIW I prefer the order you have now. I wish we had consistent tooling to do this automatically so I could stop thinking about it 😅

Reply:
Just a data point but I personally prefer the order that brooks mentioned:

core 
std
external
Internal